resources/metadata/kubernetesDeploy.yaml

```diff
@@ -231,14 +231,6 @@ spec:
           - STAGES
           - STEPS
         default: regsecret
-      - name: createDockerRegistrySecret
```

Comment:
@modulo11 Was this removal intentional? Why was it removed?

Reply:
@modulo11 Do you remember why this was removed?

Reply:
afaik this property is not used by the step